content/acra/security-controls/transport-security/acra-connector.md

```diff
@@ -3,7 +3,7 @@ title: AcraConnector
 weight: 2
 ---
 
-# AcraConnector
+# AcraConnector (deprecated since 0.91.0)
 
```

Comment:
It will be helpful to add some text that advise to use TLS and link to TLS configuration.